api/bin/www

```diff
@@ -13,7 +13,7 @@ var mongoose = require("mongoose");
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || "8080");
+var port = normalizePort(process.env.PORT || "3000");
```

Comment:
this would set the backend port to 3000 unless a different port is provided as an environment variable. the port 3000 is normally reserved for the frontend react app, is there a specific reason why we are changing the port?